web_teleop/launch/app.launch

```diff
@@ -0,0 +1,17 @@
+<launch>
+  <include file="$(find uw_gazebo)/launch/house_simulation.launch" />
+
+  <arg name="rviz" default="true"/>
+  <arg name="map_file" default="$(find uw_gazebo)/maps/house/map.yaml"/>
+
+  <include file="$(find uw_fetch_bringup)/launch/navigation.launch">
```

Comment:
There's no mechanism for rws to pass arguments to this launch file, right? If that's the case, it wouldn't make sense for it to launch navigation because it wouldn't know what map file to use.

Reply:
This is part of the `house_simulation.launch` without gazebo and I just pasted here. Because I commented out the navigation launch in [`res_fetch.launch`](https://github.com/hcrlab/rws/blob/92ef76bbaa9b96345c1ea916fab8601be65a2470/launch/rws_fetch.launch#L13-L16) so I thought this navigation should be launched somewhere? Maybe that's unnecessary for `web_teleop`.

Reply:
We shouldn't need it since the code doesn't send anything to `move_base`. Other web apps for RoboCup can expect that navigation will have been launched as part of the task or as part of the robot's bringup.